cloud/client.go

```diff
@@ -861,15 +1027,30 @@ func (c *client) WhoAmI() (string, string, bool, error) {
 	authType := "ssh"
 	if c.password != "" {
 		authType = "password"
-	} else if c.authToken != "" {
+	} else if c.authCredToken != "" {
 		authType = "token"
 	}
 
 	return pingResponse.Email, authType, pingResponse.WriteAccess, nil
 }
 
-func (c *client) getAuthTokenPath(create bool) (string, error) {
-	confDirPath := c.authTokenDir
+func (c *client) migrateV1Token() error {
+	confDirPath := c.authDir
+	if confDirPath == "" {
+		confDirPath = cliutil.GetEarthlyDir()
+	}
+	tokenPath := filepath.Join(confDirPath, "auth.token")
+	newPath := filepath.Join(confDirPath, "auth.credentials")
+	if fileutil.FileExists(tokenPath) {
+		if err := os.Rename(tokenPath, newPath); err != nil {
+			return errors.Wrapf(err, "failed to move v1 token from '%s' to '%s'", tokenPath, newPath)
```

Comment:
`v1` might confuse earthly users. Currently we only have `v0.6.4`.

Reply:
Yeah I can see that, I was thinking `v1` and `v2` versions of the underlying authentication method. I wonder if even the file name `auth.v2.token` could be confusing as well, we could just call it `auth.jwt`

Reply:
`auth.jwt` sounds like a good idea.

Reply:
Only thing I don't like is that it's the only place in the code that knows it's jwt. Otherwise, you could swap the jwt form some shiny new auth type one day and everything else here should be totally valid (except for maybe a comment or two)

Reply:
but maybe that's overly optimistic anyway, using a new auth type might not be so simple